templates/staging-test.yaml

```diff
@@ -0,0 +1,26 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
this hasn't been removed

Reply:
![image](https://user-images.githubusercontent.com/73693628/165926779-74a5467b-1014-4e1b-b754-97b1f51c808c.png)
it is added over here